l10n_ec_ote/.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
this file is not needed here, please update if necesary into root folder https://github.com/Odoo-EC/l10n-ecuador/blob/15.0/.gitignore